packages/core/src/types.ts

```diff
@@ -25,7 +25,7 @@
  *
  * @example
  * ```ts title=index.ts
- * import { Auth } from "@auth/core"
+ * import Auth from "@auth/core"
```

Comment:
Doesn't look like the `packaegs/core/src/index.ts` has a default export. I think this one was correct as-is

Reply:
Done!